modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -5,89 +5,108 @@
+3. The 2,500 pods per node tested maximum was validated on small bare metal clusters by setting `maxPods` to that value using a custom kubelet configuration. The `hostPrefix` value will need to be set to a value large enough to accommodate the number of pods.
 4. When there are a large number of active projects, etcd might suffer from poor performance if the keyspace grows excessively large and exceeds the space quota. Periodic maintenance of etcd, including defragmentation, is highly recommended to free etcd storage.
 5. There are a number of control loops in the system that must iterate over all objects in a given namespace as a reaction to some changes in state. Having a large number of objects of a given type in a single namespace can make those loops expensive and slow down processing given state changes. The limit assumes that the system has enough CPU, memory, and disk to satisfy the application requirements.
 6. Each service port and each service back-end has a corresponding entry in iptables. The number of back-ends of a given service impact the size of the endpoints objects, which impacts the size of data that is being sent all over the system.
 7. {product-title} has a limit of 512 total custom resource definitions (CRD), including those installed by {product-title}, products integrating with {product-title} and user created CRDs. If there are more than 512 CRDs created, then there is a possibility that `oc` commands requests may be throttled.
 --
-[NOTE]
-====
-Red Hat does not provide direct guidance on sizing your {product-title} cluster. This is because determining whether your cluster is within the supported bounds of {product-title} requires careful consideration of all the multidimensional factors that limit the cluster scale.
-====
+
+[id="cluster-maximums-major-releases-example-scenario_{context}"]
+== Example scenario
+
+As an example, 500 worker nodes (m5.2xl) were tested, and are supported, using {product-title} 4.12, the OVN-Kubernetes network plug-in, and the following workload objects:
+
+* 200 namespaces, in addition to the defaults
+* 60 pods per node; 30 server and 30 client pods (30k total)
+* 57 image streams/ns (11.4k total)
+* 15 services/ns backed by the server pods (3k total)
+* 15 routes/ns backed by the previous services (3k total)
+* 20 secrets/ns (4k total)
+* 10 config maps/ns (2k total)
+* 6 netpol/ns, including deny-all, allow-from ingress and intra-namespace rules
+* 57 builds/ns
+
+The following factors are known to affect cluster workload scaling, positively or negatively, and should be factored into the scale numbers when planning a deployment.  For additional information and guidance, contact your sales representative or Red Hat support.
```

Comment:
Great opportunity to `link:https://access.redhat.com/support/[Red Hat support]`.